### ISSUE TITLE
Make flannel VXLAN integration example default in the documentation

### ISSUE THREAD

**Issue by erkac** [F5Networks/f5-ci-docs#743]
Make flannel VXLAN integration example default in the documentation.

Myself and our partner spent significant time troubleshooting why the CIS and BIG-IP doesn't work as expected. We followed the configuration guide and it is misleading:
1. https://clouddocs.f5.com/containers/v2/kubernetes/kctlr-use-bigip-k8s.html shows how to configure VXLAN on BIG-IP side
2. Next button brings us to: https://clouddocs.f5.com/containers/v2/kubernetes/kctlr-app-install.html where Basic Deployment is like the one, we should use and then "a few" lines below there is reference to "Deployments for flannel BIG-IP Integrations"

If the 1) explaining the configuration using VXLANs, the very next reference should be also about flannel/vxlan integration.

Thank you.

Regards,
Lubos

**Comment by walterk82** [comment]
Agreed, step 2 at https://clouddocs.f5.com/containers/latest/userguide/cis-installation.html#creating-vxlan-tunnels

There is no reference on where profile `k8s-vxlan` is created or what it should contain for flannel.
